packages/gestalt/src/ToastComponents.js

```diff
@@ -0,0 +1,176 @@
+// @flow strict
```

Comment:
We can use nested directories now, so let's break these out into individual files in `packages/gestalt/src/Toast/`

Reply:
Lets chat about this, I dont think it works when the subfile is accessing components: Box etc. Just if it doesnt